contracts/account-history-contract/src/tests.rs

```diff
@@ -22,19 +22,22 @@ fn history() {
         Price::new("uusdc", Decimal::from_str("1.0").unwrap()),
     ];
 
-    let infos = vec![OracleAssetInfo::new(
-        "uusdc".to_string(),
-        "UUSDC".to_string(),
-        "".to_string(),
-        "".to_string(),
-        2,
-    ),OracleAssetInfo::new(
-        "uelys".to_string(),
-        "UELYS".to_string(),
-        "".to_string(),
-        "".to_string(),
-        2,
-    )];
+    let infos = vec![
+        OracleAssetInfo::new(
+            "uusdc".to_string(),
+            "UUSDC".to_string(),
+            "".to_string(),
+            "".to_string(),
+            2,
+        ),
+        OracleAssetInfo::new(
+            "uelys".to_string(),
+            "UELYS".to_string(),
+            "".to_string(),
+            "".to_string(),
+            2,
+        ),
+    ];
```

Comment:
@politeWall any reason why you still need this as you are using asset profile to retrieve decimals value?

Reply:
The reason why is to have a simpler way to mock the "getEntry" query.